duckduckgo/duckduckgo-privacy-extension
# New UX Icons

**Reviewer:** @andrey-p 



## Description:
- Add new high contrast toolbar icons
- Add new hero rating SVGs (except "null" and "calculating" special cases, that will be done in another PR)
- Add the plus icon for the enhanced grade icons, since we'll have B+ and C+ with the new algorithm, and tweak the styles accordingly


## Steps to test this PR:
1. Make sure to remove and re-install the extension (otherwise the assets might not update correctly)
2. Visit some websites and check the toolbar icon is the new high contrast one, and that it shows the right grade
3. Open the popup - the grade should match the toolbar icon
4. When protection is disabled, the hero icon should be the disabled variant (with the grey little sidebar)
5. It shouldn't yet be possible to see the new enhanced grades with the "+" icon, but they can be forced in the code by removing display:none [here](https://github.com/duckduckgo/duckduckgo-privacy-extension/compare/maria/new-assets?expand=1#diff-a7d88bc4e512a9e971f0c640bbd991e9R251)


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
